narayana-full/pom.xml

```diff
@@ -391,5 +425,64 @@
           </dependency>
       </dependencies>
     </profile>
+    <profile>
+	  <id>jdk9+</id>
+          <activation>
+             <jdk>[9,)</jdk>
+          </activation>
+          <properties>
+             <jvm.args.modular>--add-modules=java.se</jvm.args.modular>
+	     <version.org.jboss.spec.javax.xml.bind.jboss-jaxb-api_2.3_spec>1.0.1.Final</version.org.jboss.spec.javax.xml.bind.jboss-jaxb-api_2.3_spec>
+	     <version.org.jboss.spec.javax.xml.ws.jboss-jaxws-api_2.3_spec>1.0.0.Final</version.org.jboss.spec.javax.xml.ws.jboss-jaxws-api_2.3_spec>
+	     <version.jaxws-ri>2.3.0</version.jaxws-ri>
+	  </properties>
+	  <dependencies>
+	     <dependency>
```

Comment:
Is this because these are no longer in the JDK? I am not sure why these are only needed in narayana-full in order to generate the Javadoc?

Reply:
yeah, it has been removed in the JDK-11. In JDK-9 it needs to add the "--add-modules=java.xml.ws,java.xml.bind" with the javadoc options but it through the error with can not find "javax.annotation.Priority" even if the the javax.annotation-api jar is in the classpath . I think this could be a split package issue. 

The java.xml.ws depends on the java.xml.ws.annotations which contains the package "javax.annotation" but does not have the "javax.annotation.Priority". I tried to add the --patch-module  but the javadoc can not find the source files with this options.